djstripe/stripe_objects.py

```diff
@@ -68,12 +57,15 @@ class StripeObject(TimeStampedModel):
     objects = models.Manager()
     stripe_objects = StripeObjectManager()
 
+    stripe_id = StripeIdField(unique=True, stripe_name='id')
+    livemode = StripeNullBooleanField(default=False, null=True,
+                                      help_text="Null here indicates that data was unavailable. Otherwise, this field "
+                                                "indicates whether this record comes from Stripe test mode or live "
+                                                "mode operation.")
+
```

Comment:
Upgrading old records. Those will be unknown technically speaking.
On Aug 7, 2015 8:17 PM, "Alexander Kavanaugh" <notifications@github.com>
wrote:

> In djstripe/stripe_objects.py
> <https://github.com/pydanny/dj-stripe/pull/245#discussion_r36571816>:
>
> > @@ -68,12 +57,15 @@ class StripeObject(TimeStampedModel):
> >      objects = models.Manager()
> >      stripe_objects = StripeObjectManager()
> >
> > +    stripe_id = StripeIdField(unique=True, stripe_name='id')
> > +    livemode = StripeNullBooleanField(default=False, null=True,
> > +                                      help_text="Null here indicates that data was unavailable. Otherwise, this field "
> > +                                                "indicates whether this record comes from Stripe test mode or live "
> > +                                                "mode operation.")
> > +
>
> In what situation would this be null?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydanny/dj-stripe/pull/245/files#r36571816>.
>